tests/pyincore/analyses/galvestoncge/test_galvestoncge.py

```diff
@@ -0,0 +1,26 @@
+from pyincore import IncoreClient, Dataset
+from pyincore.analyses.galvestoncge import GalvestonCGEModel
+import pyincore.globals as pyglobals
+
+# This script runs SaltLakeCGEModel analysis with input files from
+# IN-CORE development services.
+
+
+def run_base_analysis():
+    client = IncoreClient(pyglobals.INCORE_API_DEV_URL)
+    galveston_cge = GalvestonCGEModel(client)
+
+    galveston_cge.set_parameter("model_iterations", 1)
+    galveston_cge.load_remote_input_dataset("SAM", "63c8398f4b843238326be519")
+    galveston_cge.load_remote_input_dataset("BB", "63a354f54b8432383268cd92")
+    galveston_cge.load_remote_input_dataset("MISCH", "63c838954b843238326be513")
+    galveston_cge.load_remote_input_dataset("EMPLOY", "63c836d94b843238326bd982")
+    galveston_cge.load_remote_input_dataset("JOBCR", "63c837f74b843238326be507")
+    galveston_cge.load_remote_input_dataset("OUTCR", "63c838414b843238326be50d")
+    galveston_cge.load_remote_input_dataset("sector_shocks", "63a358404b8432383268cee0")
```

Comment:
Can you post the local files you are using? I can give it a try. Those Ids are still pointing to SLC...

Reply:
`def run_base_analysis():
    client = IncoreClient(pyglobals.INCORE_API_DEV_URL)
    galveston_cge = GalvestonCGEModel(client)

    sam = Dataset.from_file('/Users/mohanar2/Desktop/NCSA Projects/IN-CORE/Resources/Share/Galveston_2019_CGE_Model/galveston/galveston_data/Galveston_SAM_2019.csv',
                            'incore:CGEsam')
    bb = Dataset.from_file('/Users/mohanar2/Desktop/NCSA Projects/IN-CORE/Resources/Share/Galveston_2019_CGE_Model/galveston/galveston_data/Galveston_CAPCOM_2019.csv',
                           'incore:CGEbb')
    jobcr = Dataset.from_file('/Users/mohanar2/Desktop/NCSA Projects/IN-CORE/Resources/Share/Galveston_2019_CGE_Model/galveston/galveston_data/MiscFile/Galveston_JOBCORE_2019.csv',
                              'incore:CGEjobcr')
    misch = Dataset.from_file('/Users/mohanar2/Desktop/NCSA Projects/IN-CORE/Resources/Share/Galveston_2019_CGE_Model/galveston/galveston_data/MiscFile/Galveston_HH_2019.csv',
                              'incore:CGEmisch')
    employ = Dataset.from_file('/Users/mohanar2/Desktop/NCSA Projects/IN-CORE/Resources/Share/Galveston_2019_CGE_Model/galveston/galveston_data/MiscFile/Galveston_FACTOR_2019.csv',
                               'incore:CGEemploy')
    outcr = Dataset.from_file('/Users/mohanar2/Desktop/NCSA Projects/IN-CORE/Resources/Share/Galveston_2019_CGE_Model/galveston/galveston_data/MiscFile/Galveston_OUTCR_2019.csv',
                              'incore:CGEoutcr')
    sector_shocks = Dataset.from_file('/Users/mohanar2/Desktop/NCSA Projects/IN-CORE/Resources/Share/Galveston_2019_CGE_Model/galveston/galveston_data/sector_shocks.csv',
                                      'incore:capitalShocks')

    galveston_cge.set_parameter("model_iterations", 1)
    # galveston_cge.load_remote_input_dataset("SAM", "63c8398f4b843238326be519")
    galveston_cge.set_input_dataset("SAM", sam)
    # galveston_cge.load_remote_input_dataset("BB", "63a354f54b8432383268cd92")
    galveston_cge.set_input_dataset("BB", bb)
    # galveston_cge.load_remote_input_dataset("MISCH", "63c838954b843238326be513")
    galveston_cge.set_input_dataset("MISCH", misch)
    # galveston_cge.load_remote_input_dataset("EMPLOY", "63c836d94b843238326bd982")
    galveston_cge.set_input_dataset("EMPLOY", employ)
    # galveston_cge.load_remote_input_dataset("JOBCR", "63c837f74b843238326be507")
    galveston_cge.set_input_dataset("JOBCR", jobcr)
    # galveston_cge.load_remote_input_dataset("OUTCR", "63c838414b843238326be50d")
    galveston_cge.set_input_dataset("OUTCR", outcr)
    # galveston_cge.load_remote_input_dataset("sector_shocks", "63a358404b8432383268cee0")
    galveston_cge.set_input_dataset("sector_shocks", sector_shocks)
    galveston_cge.run_analysis()


if __name__ == '__main__':
    run_base_analysis()
`

The link for the data is https://github.com/lutao11/Share/tree/main/Galveston_2019_CGE_Model/galveston/galveston_data

I have attached the testing code I use, so you just need to modify the file path :)